CONTRIBUTING.md

```diff
@@ -8,6 +8,48 @@ When you submit a pull request, a CLA bot will automatically determine whether y
 a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
 provided by the bot. You will only need to do this once across all repos using our CLA.
 
+# Types of Contributions
+
+## Submit a pull request that aligns with our goals and priorities
+The STL repo has many filed issues that track work to be completed.
+If you're unsure of where to start, you may want to start by:
+ * Looking for pinned issues
+ * Checking issues under the labels `high priority` and `help wanted`
+ * New contributors may also look into issues labeled `good first issue`
+
+## Review a pull request
+
+We welcome code reviews from contributors! Reviews from other contributors can often accelerate the reviewing process
+by helping a PR reach a more finished state before maintainers review the changes.  As a result, such a PR may require
+fewer maintainer review iterations before reaching a "Ready to Merge" state!
+To gain insight into our Code Review process, you may want to view:
+ * Ongoing reviews in [Pull Requests List](https://github.com/microsoft/STL/pulls)
+ * [Advice for Reviewing Pull Requests](https://github.com/microsoft/STL/wiki/Advice-for-Reviewing-Pull-Requests).
+ * [Code Review Videos](https://github.com/microsoft/STL/wiki/Code-Review-Videos!)
+
+# PR Checklist
+
+Before submitting a pull request, please ensure that:
+
+* Identifiers in product code changes are properly `_Ugly` as per

```

Comment:
```suggestion
1. Any non-standard identifiers in product code (including local variables, exposition-only members, etc.)
   begin with an underscore and a capital letter, as per [\[lex.name\]p3.1][]
```

grammar, make the link out-of-line, etc.

Reply:
Was not able to make it non-inline link